packages/super-components/components/msc-billing-tile/mock/handlers.ts

```diff
@@ -0,0 +1,118 @@
+import { toMswHandlers, Handler } from '../../../_common/msw-helpers';
+import vpsServiceInfos from './v6/vps-service-infos.json';
+import vpsServiceDetails from './v6/vsp-service-details.json';
+import nashaServiceInfos from './v6/nasha-service-infos.json';
+import nashaServiceDetails from './v6/nasha-service-details.json';
+import expiredServiceInfos from './v6/expired-service-infos.json';
+import expiredServiceDetails from './v6/expired-service-details.json';
+import domainServiceInfos from './v6/domain-service-infos.json';
+import domainServiceDetails from './v6/domain-service-details.json';
+import domainPropertiess from './v6/domain-properties.json';
+import contactResponse from './v6/me-contact.json';
+import deleteAtExpirationServiceInfos from './v6/delete-at-expiration-service-infos.json';
+import deleteAtExpirationServiceDetails from './v6/delete-at-expiration-service-details.json';
+import emailsServiceInfos from './v6/emails-service-infos.json';
+import emailsServiceDetails from './v6/emails-service-details.json';
+import emailsProperties from './v6/emails-properties.json';
+import hostingWebServiceInfos from './v6/hosting-web-service-infos.json';
+import hostingWebServiceDetails from './v6/hosting-web-service-details.json';
+import hostingWebProperties from './v6/hosting-web-properties.json';
```

Comment:
**suggestion:** I see that we have mock response for API placed in each super-component.
Instead of duplicating this for all components, maybe we can place it in a common folder and reuse it all components no?

Reply:
The idea is not to create a partially mocked API in each super-component
The idea is to show some relevant scenarii for each super-component

So we don't need to have the whole list of mocks to choose from in each stories or a whole mocked API to configure everytime we need to develop or test some component.

Additionally, this will make the super-components more independent and less prone to breaking since they don't depend on exterior resources.

Reply:
I get the idea. but lets say we have a scenario where super-component depends on an API `/me`. Another super-component also depends on the same API.
So, when the response of `/me` API changes, we will find ourselves in a place where we have to update the response in multiple super-components.